### ISSUE TITLE
Manage a whitelist

### ISSUE THREAD

**Issue by IOI-655321** [cowlicks/privacypossum#30]
Id love to use this but really need the ability to whitelist certain domains to de-activate the add-on.  

The ability to import/export this whitelist like Privacy Badger would be very good so i can share it between devices.

Sounds like its a nice little plugin but will hopefully mature quite fast.

**Comment by cowlicks** [comment]
Thank you for the feedback! I think this would fit best on the options page re #23

**Comment by lukf** [comment]
Yes. PP's script injection triggers my site's CSP, which is annoying for development.

**Comment by IOI-655321** [comment]
Any news on when options such as this will be in development? Im using it on my main browser but so far not using PP anywhere else due to setting up all my sites.

**Comment by cowlicks** [comment]
@IOI-655321 I'm deferring most optional user configuration until I feel privacy possum's anti-tracking heuristics are more complete. Sorry if this inconvenient. The reason is that each user configuration basically double the possible states of the program. This can make debugging and testing more-and-more difficult. Which slows the pace of development, and is always an extra consideration when adding a new feature or refactoring.

I can provide you with a quick and dirty script that would let you do this manually, would this be a sufficient solution until something more complete can be implemented?

@lukf does this still happen if you deactivate PP for your site?  This should have been fixed #35

**Comment by lukf** [comment]
@cowlicks fixed.
